porcupine/plugins/directory_tree.py

```diff
@@ -1,3 +1,9 @@
+"""
+Displays a directory tree on the left side of the editor with the folders containing the 5 most recently edited files.
```

Comment:
A couple problems with 5 most recently:
- The 5 most recent things are projects, not files. For this plugin, "project" and "directory" are slightly different concepts.
- The 5 is not true in all cases. There can be more or less. See the comment above `MAX_PROJECTS`.

I think a good solution is to just delete this:

```suggestion
Displays a directory tree on the left side of the editor.
```